scripts/clients.mk

```diff
@@ -37,10 +37,10 @@ geth-loadtest: build fund ## Run loadtest against an EVM/Geth chain.
 	sleep 5
 	$(BUILD_DIR)/$(BIN_NAME) loadtest \
 		--verbosity 700 \
+		--rpc-url http://127.0.0.1:$(PORT) \
```

Comment:
nit: potentially make host configurable

Reply:
yeah we could, this is just a handy command to trigger load test in ci :)